src/Workspaces/Core/MSBuild/Rpc/RpcClient.cs

```diff
@@ -142,7 +142,11 @@ public Task InvokeAsync(int targetObject, string methodName, List<object?> param
         using (await _sendingStreamSemaphore.DisposableWaitAsync(cancellationToken).ConfigureAwait(false))
         {
             await _sendingStream.WriteLineAsync(requestJson).ConfigureAwait(false);
+#if NET8_0_OR_GREATER
+            await _sendingStream.FlushAsync(cancellationToken).ConfigureAwait(false);
+#else
             await _sendingStream.FlushAsync().ConfigureAwait(false);
+#endif
```

Comment:
@jasonmalinowski an analyzer flagged this in `net8.0` as there is a new overload.

Reply:
I think my immediate preference would be to pass CancellationToken.None, since practically once we've started to send the request we really don't expect any further cancellation, especially per the comment in:

https://github.com/dotnet/roslyn/blob/718d6466c526309ac38b33fc0e2a209b7fd63a47/src/Workspaces/Core/MSBuild.BuildHost/Rpc/Readme.md?plain=1#L13

That said, there's an existing bug here where if we cancelled and never sent the request, we still end up leaving it in _outstandingRequests which isn't great either.

So if you can change this to CancellationToken.None prior to merge that'd be preferred, but if you can't (or there's too much blocked on this PR) I'll do a follow-up either way.

Reply:
Had to make some more changes so I went ahead and jus suppressed the warning here and filed this issue to follow up 

https://github.com/dotnet/roslyn/issues/71580